_pkgdown.yml

```diff
@@ -1,42 +0,0 @@
-url: https://extendr.github.io/
-
-title: Extendr
-
-home:
-  title: |
-    Extendr: A safe and user friendly R extension interface using Rust.
-  sidebar:
-    structure: [custom, license, dev, authors]
-    components:
-      custom:
-        title: ""
-        text: |
-          <img src="man/figures/extendr-logo-256.png" alt="Extendr logo">
```

Comment:
Can we find somewhere to put the logo please?

Reply:
done! but a little janky atm...